spec/rubocop/cop/style/braces_around_hash_parameters_spec.rb

```diff
@@ -351,6 +351,25 @@
         )
         RUBY
       end
+
+      it 'corrects when the opening brace is before the first hash element' \
+         'at same line' do
+        corrected = autocorrect_source(<<-RUBY.strip_indent)
+          foo = Foo.new(
+            { foo: 'foo',
+              bar: 'bar',
+              baz: 'this is the last element'}
+          )
+        RUBY
+
+        expect(corrected).to eq(<<-RUBY.strip_indent)
+          foo = Foo.new(
+             foo: 'foo',
```

Comment:
To correct this indentation gap, the opening brace must be replaced with space instead of being removed.

First of all, this PR solves the problem that last element is lost by auto-correct. I'm considering solving this indentation gap problem with another PR.

Until then this indetation gap will be aligned with `Layout/AlignHash` cop.